packages/dataviews/src/components/dataviews-item-actions/index.tsx

```diff
@@ -179,6 +185,13 @@ export function ActionsMenuGroup< Item >( {
 	);
 }
 
+function hasOnlyOneActionAndIsPrimary< Item >(
+	primaryActions: Action< Item >[],
+	actions: Action< Item >[]
+) {
+	return primaryActions.length === 1 && actions.length;
```

Comment:
@oandregal I believe the condition `actions.length` should be updated to `actions.length === 1`. The current implementation only displays the primary action when there is exactly one primary action, even if additional non-primary actions are present.

https://wordpress.github.io/gutenberg/?path=/story/dataviews-dataviews--default
<img width="2247" alt="Screenshot 2024-11-21 at 17 01 42" src="https://github.com/user-attachments/assets/b0ce37e9-780b-4e4a-8ec3-135104382976">
https://github.com/okmttdhr/gutenberg/blob/b7d989e0dafb23866f65295a851e6b8734c3b693/packages/dataviews/src/components/dataviews/stories/fixtures.tsx#L610